blocks/banner/banner.js

```diff
@@ -0,0 +1,75 @@
+import {
+  div, button, h2, h3, form, input, a,
+} from '../../scripts/dom-helpers.js';
+
+function createLinkElement(hrefValue, innerHTML) {
```

Comment:
is there any value with this function or is it simple enough that it could be replaced by the caller with the call to 

```a({ class: 'spotlight-link', href: hrefValue }, innerHTML);```

It would essentially remove a function invocation.